.github/dependabot.yml

```diff
@@ -11,3 +11,13 @@ updates:
     directory: "/"
     schedule:
       interval: "weekly"
+
+  - package-ecosystem: "terraform"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+
+  - package-ecosystem: "terraform"
+    directory: "/examples/basic_usage"
+    schedule:
+      interval: "weekly"
```

Comment:
```suggestion
```

Reply:
This repo has such a directory, does it not?

Reply:
See [here](https://github.com/cisagov/cool-configure-aws-account/pull/2#pullrequestreview-850679218) for the answer as to why @dav3r (correctly) made this change.